deps/node-inspect/.eslintrc

```diff
@@ -127,21 +127,3 @@ rules:
 
 # Global scoped method and vars
 globals:
-  COUNTER_HTTP_CLIENT_REQUEST: false
-  COUNTER_HTTP_CLIENT_RESPONSE: false
-  COUNTER_HTTP_SERVER_REQUEST: false
-  COUNTER_HTTP_SERVER_RESPONSE: false
-  COUNTER_NET_SERVER_CONNECTION: false
-  COUNTER_NET_SERVER_CONNECTION_CLOSE: false
-  DTRACE_HTTP_CLIENT_REQUEST: false
-  DTRACE_HTTP_CLIENT_RESPONSE: false
-  DTRACE_HTTP_SERVER_REQUEST: false
-  DTRACE_HTTP_SERVER_RESPONSE: false
-  DTRACE_NET_SERVER_CONNECTION: false
-  DTRACE_NET_STREAM_END: false
-  LTTNG_HTTP_CLIENT_REQUEST: false
-  LTTNG_HTTP_CLIENT_RESPONSE: false
-  LTTNG_HTTP_SERVER_REQUEST: false
-  LTTNG_HTTP_SERVER_RESPONSE: false
-  LTTNG_NET_SERVER_CONNECTION: false
-  LTTNG_NET_STREAM_END: false
```

Comment:
Isn't this file handled upstream somewhere?

Reply:
Yeah, it comes from here: https://github.com/nodejs/node-inspect/blob/master/.eslintrc. I think either way is fine (remove it here and backport or open an issue to remove it upstream and then wait for it to make its way into core).

Reply:
nah, @BridgeAR is right, our policy is to change these upstream first, I just hadn't noticed that it was a deps file